sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -2498,6 +2498,90 @@ matrix(
           assert.equal(pollCount, implName === "createPoller" ? 2 : 1);
         });
       });
+      describe("general behavior", function () {
+        it("poll() doesn't poll after the poller is in a succeed status", async function () {
+          const poller = await createTestPoller({
+            routes: [
+              {
+                method: "PUT",
+                status: 200,
+                body: `{ "properties": { "provisioningState": "Succeeded"}, "id": "100", "name": "foo" }`,
+              },
+            ],
+            throwOnNon2xxResponse,
+          });
+          const result = await poller.pollUntilDone();
+          await poller.poll(); // This will fail if a polling request is sent
```

Comment:
Not sure this is necessary. Besides this case could we add a test case with below steps which are exactly aligned with SDK?

```
- const poller = await createTestPoller
- await poller.poll()
- const result = await poller.pollUntilDone()
```

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/24159/commits/4449d5f9077bd8f3908fd2d0a35e0a152da8ff6d